### DISCUSSION TITLE
Epiverse-TRACE developer space - Communicating development stages of open-source software

### DISCUSSION THREAD

**Discussion by giscus** [epiverse-trace/epiverse-trace.github.io#78] [Blog post discussion]
# Epiverse-TRACE developer space - Communicating development stages of open-source software

A place for Epiverse-TRACE developers to share their reflections, learnings, and showcase their work.

https://epiverse-trace.github.io/posts/comm-software-devel/

**Comment by avallecam** [comment]
Nice review on the topic! For non-package repositories I'm currently using the [`{badger}`](https://github.com/GuangchuangYu/badger) package to generate the text for them.
